<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobiiContactPanel</class>
 <widget class="QWidget" name="MobiiContactPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>724</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>408</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MobiiContactPanel</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="contactsListContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLineEdit" name="searchContact">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"> border: 2px solid #008287;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Tapez ici pour rechercher dans vos contacts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newContact">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Ajouter un contact</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="mobii.qrc">
            <normaloff>:/buttons/Resources/add.png</normaloff>:/buttons/Resources/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="contactList">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="contactContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>711</width>
           <height>577</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="styleSheet">
             <string notr="true"> border: 1px solid #008287;
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="buttonsLayout" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="getContactButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>Synchroniser les contacts</string>
           </property>
           <property name="icon">
            <iconset resource="mobii.qrc">
             <normaloff>:/buttons/Resources/sync.png</normaloff>:/buttons/Resources/sync.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendContactsToMobile">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>Envoyer les modifications
vers mon mobile</string>
           </property>
           <property name="icon">
            <iconset resource="mobii.qrc">
             <normaloff>:/buttons/Resources/share.png</normaloff>:/buttons/Resources/share.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mobii.qrc"/>
 </resources>
 <connections/>
</ui>
